media-libs/libgfx/libgfx-1.1.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Probably we can already use `EAPI=8` on this ebuild

Reply:
I changed the EAPI to 8 in the commit https://github.com/gentoo/sci/pull/1100/commits/d4b4ded1e25e5090e9a612b9db307e16a432b1b9